app/gui/view/src/project.rs

```diff
@@ -301,12 +304,10 @@ impl Model {
     }
 
     fn set_light_style(&self) {
-        ensogl_hardcoded_theme::builtin::light::enable(&self.app);
         self.set_html_style("light-theme");
     }
```

Comment:
1. We remove `enable` call everywhere because now it is called in a before-main entry point, next to where themes are defined.
2. We still need to call `setlight_theme` to propagate the styles to HTML elements.
3. That's true, calling `set_dark_theme` will change only HTML styles, but we have comments about it so I find it OK.

Is it OK for you or you want to change it? If the latter, how would you like to do it?

Reply:
So, either I would still call enable here (and in set_dark_stype), so it will work at least to some extent, or remove _all_ the code for setting styles, including set_style method here.

Reply:
But if you remove all the code, HTML will not be properly styled.

Reply:
Then take the first option. set styles not only on html, but also on these parts of ensogl which would work (actually, all which read styles from FRP outside shaders).